### PR TITLE
[#161966904 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by codeplus254**
#### What does this PR do?
Integrates TravisCI badge onto the readme. 

#### Description of Task to be completed?
Add .travis.yml file into the root folder of the project.

#### How should this be manually tested?

- Go to [Github Repo](https://github.com/andela/ah-infinity-stones)

- Select `ch-integrate-travisci-readme-161966904` branch

- Click the TravisCI badge. This will redirect you to the travis.org webpage with Andela/

#### What are the relevant pivotal tracker stories?
[#161966904](https://www.pivotaltracker.com/story/show/161966904)

### CODE REVIEW

**Comment by r-wambui** [comment]
Thank you. I have added it

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- postgresql
+
+addons:
+ postgresql: '9.6'
+
+# install dependencies
+install:
+- pip3 install -r requirements.txt
+
+#before_script:
```
Comment (r-wambui) [inline .travis.yml]: Kindly do not push commented code. Check from line 19 -26.

**Comment by r-wambui** [comment]
@codeplus254 You do not have to *force push* every time, Just do a normal  push.

**Review by r-wambui — COMMENTED** [review]
> Thank you. I have added it

Please do not edit the comments given reply instead. This makes it easier to follow. I understand you had the admin rights.

File: .travis.yml (hunk 1 above)
Comment (codeplus254) [reply .travis.yml]: Oh my! Sorry about that. My intention was to reply. Okay I will remove the commented code 👍

Comment (codeplus254) [reply .travis.yml]: Hey, I squashed the commits. I believe that the commented code was a previous commit. Currently, I do not have commented codes.

**Comment by r-wambui** [comment]
@codeplus254 Please ensure that you badge on readme redirects to the project on travis.

**Comment by codeplus254** [comment]
Sure. Let me change the url

**Comment by codeplus254** [comment]
It turns out something was missing in that URL. 

PS: I have force pushed the changes since the normal push is rejected by GitHub. 👍

**Comment by ThaDeveloper** [comment]
@codeplus254 Good work on the travis setup. Please check the first 2 lines of the PR, they are out of context. You can give a title, also I don't think [Finishes ##161966904] is necessary.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- "3.6"
+# setup environment
+env:
+- DJANGO_VERSION=2.1.3
+
+services:
+- postgresql
+
+addons:
+ postgresql: '9.6'
```
Comment (amutava) [inline .travis.yml:12]: why do you have posgresql?was it part of the story

Comment (codeplus254) [reply .travis.yml:12]: I thought we would use Postgres for the backend? Should I remove it? The story was simply integrating Travis to our project. The specifics were left out.

**Pull request merged**